codespell_lib/data/dictionary.txt

```diff
@@ -5243,12 +5323,15 @@ brockened->broken
 brocolee->broccoli
 brocoli->broccoli
 brodcast->broadcast
+broge->brogue
```

Comment:
I have just learned `brogues` can be shoes, formerly worn in parts of Ireland and the Scottish Highlands.

Reply:
Thanks, now I've also learned that. I'm sure someone out there will make good use of this correction. ;-)

I've added the plural form `broges->brogues` too.